steps/aws/s3/session_test.go

```diff
@@ -67,11 +68,6 @@ func TestNewS3Session(t *testing.T) {
 		newS3SessionErr error
 		wantErr         bool
 	}{
-		{
-			name:            "New session error",
-			newS3SessionErr: fmt.Errorf("new s3 session error"),
-			wantErr:         true,
-		},
```

Comment:
This tests was not testable?

Reply:
Session configuration changes a bit with the new sdk version and client creation does not returns error. The only way to get error in this function is when LoadDefaultConfig fails but not easy to achieve without mock. 
I think that in this case we could skip this error check.